hangman/lib/game/game.ex

```diff
@@ -0,0 +1,38 @@
+defmodule Hangman.Game do
+  @moduledoc """
+  Implemnation of Hangman API.
+  """
+
+  #file with helper methods
+  import Hangman.Assist
+
+  #state structure
+  defstruct turns_left: 7,
+            game_state: :initializing,
+            letters: [],
+            used: [],
+            word: "",
+            last_guess: ""
+  
+  #new_game() method that takes a random word and uses it  
+  def new_game() do
+    word = Dictionary.random_word()
+    %Hangman.Game{}
```

Comment:
## -2 idiomatic

It would be clearer (and easier) to have

~~~ elixir
%Hangman.Game{
  word: word,
  letters:  List.duplicate....
}
~~~